creativecloud/features/changeBg/changeBg.js

```diff
@@ -52,7 +54,13 @@ function createGroups(vp, current, swatchArr, srcArr) {
   customElem.config[vp].groups.push(obj);
 }
 
-export default function changeBg(el) {
+export default async function changeBg(el) {
+  const { getConfig } = await import(`${getLibs()}/utils/utils.js`);
+  const config = getConfig();
+  if (window.location.host !== config.prodDomains[0]) {
```

Comment:
Lets restrict it for page for previewing as authoring feedback can be shared during initial stage itself and we do not need it when they are ready to publish.